src/Headless/Avalonia.Headless.Vnc/Avalonia.Headless.Vnc.csproj

```diff
@@ -1,13 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-      <TargetFrameworks>$(AvsCurrentTargetFramework);$(AvsLegacyTargetFrameworks);netstandard2.0</TargetFrameworks>
+      <TargetFrameworks>$(AvsCurrentTargetFramework);$(AvsLegacyTargetFrameworks)</TargetFrameworks>
```

Comment:
Removing a TFM is a minor-ish breaking change and the general policy is  avoid those.

Check if you can get desired results with a version supporting netstandard2.0

Reply:
Specifically 
https://www.nuget.org/packages/Quamotion.RemoteViewing/1.1.179

Reply:
OK，I will try it.

Reply:
1.1.179 works fine. I will try downgrade to this version.